modules/c++/mt/include/mt/ThreadGroup.h

```diff
@@ -89,13 +85,31 @@ class ThreadGroup
      */
     void joinAll();
 
+    /*!
+     * Checks whether CPU pinning is set (i.e. whether mAffinityInit is NULL)
+     *
+     * \returns true if CPU pinning is set, false otherwise
+     */
+    bool isPinToCPUEnabled() const;
+
+    static bool getDefaultPinToCPU();
+
+    static void setDefaultPinToCPU(const bool newDefault);
```

Comment:
I love `const` in general but don't use it when passing in variables by value into functions (reasoning being that at the caller level, they should know that the function can't change incoming variables passed by value in a way that affects them, so it doesn't improve interface readability... it does technically inform someone looking at the implementation of the method that the variable isn't going to be changed, but style-wise we're saying that the clutter of the extra `const` outweighs that benefit)